tools/cz-config.js

```diff
@@ -3,22 +3,56 @@
 module.exports = {
 
   types: [
```

Comment:
NOTE: All the extra white space within the strings is needed to align the indentation when output into the cli